### PR TITLE
Bump all of the versions to 0.2.0.0

### PR DESCRIPTION
**Pull request by fkm3**
https://github.com/tensorflow/haskell/issues/193

### CODE REVIEW

File: tensorflow-logging/tensorflow-logging.cabal [hunk 1]
```diff
@@ -29,10 +29,10 @@ library
...
                 , stm-conduit
-                , tensorflow == 0.1.*
-                , tensorflow-core-ops == 0.1.*
-                , tensorflow-ops == 0.1.*
-                , tensorflow-proto == 0.1.*
+                , tensorflow == 0.2.*
+                , tensorflow-core-ops == 0.2.*
+                , tensorflow-ops == 0.2.*
+                , tensorflow-proto == 0.2.*
                 , tensorflow-records-conduit
```
Comment (judah) [inline tensorflow-logging/tensorflow-logging.cabal]: While we're here, add an explicit version for tensorflow-records-conduit?

File: tensorflow-mnist/tensorflow-mnist.cabal [hunk 2]
```diff
@@ -28,8 +28,8 @@ library
                 , lens-family
                 , containers
                 , split
-                , tensorflow-proto == 0.1.*
-                , tensorflow-core-ops == 0.1.*
+                , tensorflow-proto == 0.2.*
+                , tensorflow-core-ops == 0.2.*
                 , tensorflow
```
Comment (judah) [inline tensorflow-mnist/tensorflow-mnist.cabal]: Add a bound for `tensorflow`?

File: tensorflow-logging/tensorflow-logging.cabal (hunk 1 above)
Comment (fkm3) [reply tensorflow-logging/tensorflow-logging.cabal]: Done.

File: tensorflow-mnist/tensorflow-mnist.cabal (hunk 2 above)
Comment (fkm3) [reply tensorflow-mnist/tensorflow-mnist.cabal]: Done.

**Pull request merged**